### ISSUE TITLE
Doesn't build 

### ISSUE THREAD

**Issue by marcdownie** [StochasticTinkr/kotlin4vulkan#1]
Fresh clone (at 5cbaf69), on aarch64:

```
$ ./gradlew build

> Task :generate:downloadVulkanDocs
Downloading Vulkan docs: -1 bytes to /Users/marc/temp/kotlin4vulkan/generate/build/vulkan-docs.zip

> Task :generate:generateClasses
Generating bindings for vulkan VK_VERSION_1_3 in /Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin from /Users/marc/temp/kotlin4vulkan/generate/build/vulkan-docs/Vulkan-Docs-1.3.288
Processing types...
(   1/1965): Processing type INCLUDE:      vk_platform
(   2/1965): Processing type DEFINE:       VK_DEFINE_HANDLE
(   3/1965): Processing type DEFINE:       VK_USE_64_BIT_PTR_DEFINES
(   4/1965): Processing type DEFINE:       VK_DEFINE_NON_DISPATCHABLE_HANDLE

[...]
WARNING: Unsupported output type `class org.lwjgl.PointerBuffer` for vkGetMemoryWin32HandleNV
        at com.stochastictinkr.vulkan.playground.generator.HandleGenerator.singleOutputCommandOverloads(HandleGenerator.kt:232)
WARNING: Unsupported output type `class org.lwjgl.PointerBuffer` for vkGetMemoryWin32HandleKHR
        at com.stochastictinkr.vulkan.playground.generator.HandleGenerator.singleOutputCommandOverloads(HandleGenerator.kt:232)
WARNING: Unsupported output type `class org.lwjgl.PointerBuffer` for vkGetSemaphoreWin32HandleKHR
        at com.stochastictinkr.vulkan.playground.generator.HandleGenerator.singleOutputCommandOverloads(HandleGenerator.kt:232)
WARNING: Unsupported output type `class org.lwjgl.PointerBuffer` for vkGetFenceWin32HandleKHR
        at com.stochastictinkr.vulkan.playground.generator.HandleGenerator.singleOutputCommandOverloads(HandleGenerator.kt:232)

[... many more 'Processing type DEFINE' ...]

e: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkAccelerationStructureBuildGeometryInfoKHR.kt:96:17 Unresolved reference: VkBuildAccelerationStructureFlagsKHR
e: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkAccelerationStructureBuildGeometryInfoKHR.kt:100:75 Unresolved reference: VkBuildAccelerationStructureFlagsKHRBuilder
e: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkAccelerationStructureBuildGeometryInfoKHR.kt:100:125 Unresolved reference: VkBuildAccelerationStructureFlagsKHR
e: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkAccelerationStructureBuildGeometryInfoKHR.kt:102:30 Unresolved reference: VkBuildAccelerationStructureFlagsKHRBuilder

[.. and so on for around 1000 error lines ..]

: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkXlibSurfaceCreateInfoKHR.kt:91:30 Unresolved reference: VkXlibSurfaceCreateFlagsKHRBuilder
e: file:///Users/marc/temp/kotlin4vulkan/build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/VkXlibSurfaceCreateInfoKHR.kt:94:56 Unresolved reference: VkXlibSurfaceCreateFlagsKHR

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

```

Those missing references are not spurious, there are only 1436 files in `./build/generated-sources/src/main/kotlin/com/stochastictinkr/vulkan/`. 

---

```
./gradlew javaToolchains

> Task :javaToolchains

 + Options
     | Auto-detection:     Enabled
     | Auto-download:      Enabled

 + Eclipse Temurin JDK 21.0.4+7-LTS
     | Location:           /Users/marc/.gradle/jdks/eclipse_adoptium-21-aarch64-os_x/jdk-21.0.4+7/Contents/Home
     | Language Version:   21
     | Vendor:             Eclipse Temurin
     | Architecture:       aarch64
     | Is JDK:             true
     | Detected by:        Auto-provisioned by Gradle

[... but I think that's the toolchain that gradlew is using ...]
```

**Comment by StochasticTinkr** [comment]
Thanks, I found this issue today too.  I refactored the way that Flags/FlagBit classes were built, and it stopped generated the Flags classes.  I didn't catch it earlier, because the generated code was still in place, but a clean build reproduced it.

There should be a new branch called `documentation` that should build now, but keep in mind that branch is a WIP.   I'll try to backport the fix to the main branch soon.
